cwms-radar-client/src/test/resources/radar/json/timeseries.json

```diff
@@ -0,0 +1 @@
0,265.502136,0],[1516539600000,265.502136,0],[1516543200000,265.505184,0],[1516546800000,265.502136,0],[1516550400000,265.502136,0],[1516554000000,265.502136,0],[1516557600000,265.505184,0],[1516561200000,265.505184,0],[1516564800000,265.505184,0],[1516568400000,265.50823199999996,0],[1516572000000,265.50823199999996,0],[1516575600000,265.505184,0],[1516579200000,265.50823199999996,0],[1516582800000,265.505184,0],[1516586400000,265.502136,0],[1516590000000,265.502136,0],[1516593600000,265.502136,0],[1516597200000,265.502136,0],[1516600800000,265.502136,0],[1516604400000,265.499088,0],[1516608000000,265.499088,0],[1516611600000,265.502136,0],[1516615200000,265.499088,0],[1516618800000,265.49604,0],[1516622400000,265.499088,0],[1516626000000,265.49604,0],[1516629600000,265.499088,0],[1516633200000,265.49604,0],[1516636800000,265.49604,0],[1516640400000,265.499088,0],[1516644000000,265.49604,0],[1516647600000,265.499088,0],[1516651200000,265.49604,0],[1516654800000,265.502136,0],[1516658400000,265.505184,0],[1516662000000,265.502136,0],[1516665600000,265.49604,0],[1516669200000,265.499088,0],[1516672800000,265.499088,0],[1516676400000,265.49604,0],[1516680000000,265.49604,0],[1516683600000,265.49604,0],[1516687200000,265.49604,0],[1516690800000,265.49604,0],[1516694400000,265.49604,0],[1516698000000,265.49604,0],[1516701600000,265.49604,0],[1516705200000,265.49604,0],[1516708800000,265.49604,0],[1516712400000,265.492992,0],[1516716000000,265.49604,0],[1516719600000,265.49604,0],[1516723200000,265.49604,0],[1516726800000,265.49604,0],[1516730400000,265.49604,0],[1516734000000,265.49604,0],[1516737600000,265.499088,0],[1516741200000,265.499088,0],[1516744800000,265.499088,0],[1516748400000,265.499088,0],[1516752000000,265.49604,0],[1516755600000,265.49604,0],[1516759200000,265.49604,0],[1516762800000,265.492992,0],[1516766400000,265.492992,0],[1516770000000,265.492992,0],[1516773600000,265.492992,0],[1516777200000,265.492992,0],[1516780800000,265.489944,0],[1516784400000,265.489944,0],[1516788000000,265.492992,0],[1516791600000,265.492992,0],[1516795200000,265.492992,0],[1516798800000,265.486896,0],[1516802400000,265.489944,0],[1516806000000,265.492992,0],[1516809600000,265.489944,0],[1516813200000,265.492992,0],[1516816800000,265.492992,0],[1516820400000,265.492992,0],[1516824000000,265.492992,0],[1516827600000,265.49604,0],[1516831200000,265.49604,0],[1516834800000,265.492992,0],[1516838400000,265.492992,0],[1516842000000,265.492992,0],[1516845600000,265.492992,0],[1516849200000,265.489944,0],[1516852800000,265.489944,0],[1516856400000,265.489944,0],[1516860000000,265.489944,0],[1516863600000,265.489944,0],[1516867200000,265.489944,0],[1516870800000,265.489944,0],[1516874400000,265.489944,0],[1516878000000,265.486896,0],[1516881600000,265.486896,0],[1516885200000,265.486896,0],[1516888800000,265.489944,0],[1516892400000,265.486896,0],[1516896000000,265.489944,0],[1516899600000,265.492992,0],[1516903200000,265.489944,0],[1516906800000,265.489944,0]]}
```

Comment:
I politely ask this gets run through a pretty-printer.... also same comment as the location, make sure the full content type is clearly identified.

Reply:
I'm not sold on the idea of pretty-printing these testing resource files. They aren't meant to be human readable as that's what the swagger docs and postman output are for, rather they should be used for rapid testing input. As this project becomes bigger, there should be more and more of these large json files, most of which is repeated data points, which isn't useful to read in VCS. 

As far as versioning in the path goes, this makes sense to me so that we don't have a diff revision history of time series.json for v2 and v3 as the API moves forward. This seems to further my stance on not pretty-printing the json.

Reply:
well, we'll see what happens the first time you need to dig into it to see what's going wrong.